Makefile

```diff
@@ -136,10 +136,6 @@ else
 	cd -
 endif
 
-.PHONY: generate-doc
```

Comment:
this "generate-doc" make target isn't used for quite some time, as this documentation is now generated from the operator repo